### DISCUSSION TITLE
Todo sobre Dashboard

### DISCUSSION THREAD

**Discussion by llGanhzll** [TheTiket/issues#22] [Ideas]
# DESCRIPTION
Abro esta discusión para que veamos temas relacionados a dashboard para futuras versiones.

**Comment by llGanhzll** [comment]
## ✨ Template para clientes nuevos

Opino que cuando se crea un cliente debe tener un Template de dashboard por defecto o en dado caso tengamos una serie de templates que el usuario pueda escoger con cual quiere iniciar a trabajar en esa vista.

**Reply by Crsk** [reply]
Dejo una captura de uno que vi y me gustó para que busquemos inspiración:

<img width="512" alt="image" src="https://user-images.githubusercontent.com/6186848/161645342-1239af40-88a5-4884-ace9-0494e22b2bc3.png">

Nótese que en pocos clicks se puede configurar tanto el layout como las características del contenido (charts)
